src/devtools/client/themes/webconsole.css

```diff
@@ -1071,7 +1061,7 @@ a.learn-more-link.webconsole-learn-more-link {
 }
 
 .webconsole-output .overlay-container:not(.debug):hover .button {
-  background: linear-gradient(152.2deg, #67daff 6.75%, #4382ff 93.14%);
+  background: linear-gradient(to bottom right, #68DCFC, #4689F8);
```

Comment:
I think we settled on a solid colour rather than a gradient:

<img width="177" alt="image" src="https://user-images.githubusercontent.com/9154902/131399158-d39b36a1-c096-47af-8ba7-434b7ff670bb.png">

If so, it should be our primary accent colour.